auth-testing/README.md

```diff
@@ -0,0 +1,3 @@
+# Auth Testing
+
+Various data simulating an OIDC config and JWKS URI
```

Comment:
I was thinking we should reference _where_ they're used as well, since otherwise I might just search this repo and say "oh, looks like these aren't being used anymore" and delete them.

```suggestion
Various data simulating an OIDC config and JWKS URI, used by the e2e-tests in api
```